client/src/partials/inventory/groups/groups.html

```diff
@@ -0,0 +1,58 @@
+<!-- breadcrumb header -->
+<bh-breadcrumb
+  path="InventoryCtrl.bcPaths"
+  button="InventoryCtrl.bcButtons">
+</bh-breadcrumb>
+
+<div class="flex-content">
+  <div class="container">
+
+    <div class="panel panel-primary">
+
+      <div class="panel-heading">
+        <i class="fa fa-list"></i>
+        {{ 'INVENTORY.LIST_GROUP' | translate }} ({{ InventoryCtrl.groupList.length }})
+      </div>
+
+      <table class="table table-condensed">
+        <thead>
+          <tr>
+            <th>{{ 'FORM.LABELS.CODE' | translate }}</th>
+            <th>{{ 'FORM.LABELS.NAME' | translate }}</th>
+            <th>{{ 'INVENTORY.CONTAINS' | translate }}</th>
+            <th class="text-center" colspan="3">{{ 'FORM.LABELS.ACCOUNT' | translate }}</th>
+            <th>&nbsp;</th>
+          </tr>
+          <tr>
+            <th></th><th></th><th></th>
+            <th class="text-center"><small>{{ 'FORM.LABELS.SALE' | translate }}</small></th>
```

Comment:
On thing to experiment with is `<th rowspan="2"></th>` to make muli-line column headers.

Here is a screenshot of your code:
![original](https://cloud.githubusercontent.com/assets/896472/16008264/0ac58db0-316f-11e6-8453-b0f7ec713d39.png)

And here is what I achieved using `rowspan="2"`.
![rowspan 2](https://cloud.githubusercontent.com/assets/896472/16008273/1ceea080-316f-11e6-91cc-0eb8b7990580.png)

I don't think mine is better (there is a lot of whitespace), but you might try messing around with it to see what you can achieve.